### PR TITLE
Cumulative multi-PR

### PR DESCRIPTION
**Pull request by jaimergp**
## Description

All PRs have been merged into this multi-merge branch before we integrate everything into `master`.

Supersedes #3 #4 #6.

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [x] Unit tests running for every module
  - [x] Documentation is building for every module

## Status
- [ ] Ready to go

### CODE REVIEW

**Comment by jaimergp** [comment]
How cool is this now, huh?

```
$> structuralalignment --method=theseus 4u3y 4u40
   _____ _                   _                   _
  / ____| |                 | |                 | |
 | (___ | |_ _ __ _   _  ___| |_ _   _ _ __ __ _| |
  \___ \| __| '__| | | |/ __| __| | | | '__/ _` | |
  ____) | |_| |  | |_| | (__| |_| |_| | | | (_| | |
 |_____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_|
     /\   | (_)                                | |
    /  \  | |_  __ _ _ __  _ __ ___   ___ _ __ | |_
   / /\ \ | | |/ _` | '_ \| '_ ` _ \ / _ \ '_ \| __|
  / ____ \| | | (_| | | | | | | | | |  __/ | | | |_
 /_/    \_\_|_|\__, |_| |_|_| |_| |_|\___|_| |_|\__|
                __/ |
               |___/

 v0.0.1 · Brought to you by @volkamerlab

RMSD for alignment between `4u3y` and `4u40` is 3.288Å
```

File: structuralalignment/superposition/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+"""
```
Comment (pipaj97) [inline structuralalignment/superposition/__init__.py:1]: Do we need that file?

File: structuralalignment/superposition/theseus.py [hunk 2]
```diff
@@ -1,5 +1,233 @@
...
+        self.alignment_app = "muscle"
+
+    def _create_pdb(self, structures):
+        """
+        Create files with atomium models
+
+        Parameter
+        ---------
+        structures : list
+            list of atomium models
```
Comment (pipaj97) [inline structuralalignment/superposition/theseus.py:77]: the return is missing in the docstring

File: structuralalignment/superposition/__init__.py (hunk 1 above)
Comment (jaimergp) [reply structuralalignment/superposition/__init__.py:1]: Unfortunately yes. It marks a directory as a (sub)package and allows us to document the whole subpackage through the module docstring, if needed.

File: structuralalignment/superposition/theseus.py (hunk 2 above)
Comment (jaimergp) [reply structuralalignment/superposition/theseus.py:77]: Nice catch!

**Pull request merged**